src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-mongodb-rbac_scenario.py

```diff
@@ -0,0 +1,104 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+class Cosmosdb_previewMongodbRbacScenarioTest(ScenarioTest):
```

Comment:
Can you update this file name `test_cosmosdb-mogodb-rbac_scenario.py` to `test_cosmosdb_mogodb_rbac_scenario.py`? The other two existing test files should also be renamed so that our CI can auto-detect these tests and help check.